packages/manager/modules/manager-components/src/region/service.js

```diff
@@ -1,6 +1,7 @@
 import trim from 'lodash/trim';
+import { COUNTRIES } from './constants';
 
-export default class CucRegionService {
+export default class ovhManagerRegionService {
```

Comment:
👍  great initiative. I have created a reusable region formater/directive (https://github.com/ovh/manager/pull/5234). Do you think we can still keep it and may be refactor it to use this region service?

Reply:
@radireddy Yes I suppose we better reuse existing code than duplicating this kind of logic (purpose here is to move so we can use it in `feat/netapp` instead of reimplementing a region logic)